Example/Database/Tests/Helpers/FTestHelpers.m

```diff
@@ -55,7 +60,9 @@ + (NSArray*) getRandomNodes:(int)num persistence:(BOOL)persistence {
 
     NSMutableArray *refs = (persistence) ? persistenceRefs : noPersistenceRefs;
 
-    id<FAuthTokenProvider> authTokenProvider = [FAuthTokenProvider authTokenProviderForApp:[FIRApp defaultApp]];
+    id<FAuthTokenProvider> authTokenProvider =
+        [FAuthTokenProvider authTokenProviderWithAuthInterop:
```

Comment:
Same as the other comment - 
"It would be nice to avoid the default app here, since we've seen it cause some issues with flakiness previously. Luckily, the container shouldn't even be needed: there's an auth interop fake [`FIRAuthInteropFake`](https://github.com/firebase/firebase-ios-sdk/blob/master/Example/Shared/FIRAuthInteropFake.h) that simulates Auth functionality - take a look at the storage examples [here](https://github.com/firebase/firebase-ios-sdk/blob/7e5befc30356604a9f324bb74664a970d148f493/Example/Storage/Tests/Unit/FIRStorageTokenAuthorizerTests.m#L35)."

Reply:
Done